### ISSUE TITLE
Home button Is not working when level change

### ISSUE THREAD

**Issue by grawish** [untitled-team-101/Bricks-Breaker#5]
![image](https://user-images.githubusercontent.com/41504211/193655187-cb7c124c-2466-415f-9e26-4c6769f5184f.png)

**Comment by aazam-gh** [comment]
hi can i work on this?

**Comment by grawish** [comment]
@Alcadeus0 Surely you can!

**Comment by aazam-gh** [comment]
Does the home button have any event handler attached to it? So far I havent found any. Should it point the user back to the first modal?

**Comment by grawish** [comment]
> Does the home button have any event handler attached to it? So far I haven't found any. Should it point the user back to the first modal?

yes, it should redirect to first modal and if you don't find any event listener then we can create one `onclick` function

**Comment by aazam-gh** [comment]
so i tried redirecting it to the first modal but then I can't seem to get the game to play again once the user clicks on all 5 modals again

**Comment by grawish** [comment]
@KarthikS373 do you think you can do this too?

**Comment by KarthikS373** [comment]
> @KarthikS373 do you think you can do this too?

How would you like to see the home button working. I mean do you want the user to be redirected to dialog page on the initial screen by saving the score or something else. Please provide a small description over it and I would love to work upon this too
